_episodes/028-rust-1.44-1.45.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "What's New in Rust 1.44 and 1.45"
+date: 2015-05-15T16:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e027-rust-1.44-1.45.mp3
+duration: "46:56"
+length: "33787656"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Jon and Ben examine the features of Rust 1.44 and Rust 1.45.
+
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+#### [@01:21] - Rust 1.44
+
+ - [@01:50] - [`cargo tree` subcommand](https://github.com/rust-lang/rust/blob/master/RELEASES.md#cargo-2)
```

Comment:
I think the `cargo-2` anchor will actually stop working — it's just a link to the second title in the document labeled Cargo, which changes over time. Maybe link to [the PR](https://github.com/rust-lang/cargo/pull/8062/) instead?